src/packages/excalidraw/README.md

```diff
@@ -601,7 +601,7 @@ getSyncableElements(elements:  <a href="https://github.com/excalidraw/excalidraw
 import { getSyncableElements } from "@excalidraw/excalidraw";
 ```
 
-This function returns all the deleted elements of the scene.
+This function returns all the elements including the deleted ones and excluding the elements which are are invisibly small (eg width & height are zero) hence excluded. This can be helpful when you want to sync the elements during collaboration.
```

Comment:
```suggestion
This function returns all the elements including the deleted ones, excluding elements which are are invisibly small (e.g. width & height are zero). Useful when you want to sync the elements during collaboration.
```

While `e.g.` and `eg` can be used both, we're using `e.g.` elsewhere so keep it consistent. I prefer `e.g.` because it's immediately obvious what you're reading, especially when it's in the middle of a sentence.